streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedStream.java

```diff
@@ -223,7 +223,9 @@
      * @param windows   the specification of the aggregation {@link Windows}
      * @return a windowed {@link KTable} that contains "update" records with unmodified keys and {@link Long} values
      * that represent the latest (rolling) count (i.e., number of records) for each key within a window
+     * @deprecated use {@code windowedBy(windows).count()}
```

Comment:
We should use `@link` instead of `@code` IMHO. Similar below.

Reply:
i'm not sure `{@link #windowedBy(windows).count()}` works?

Reply:
That would not work. A link can only point to one place. I would just link to `windowedBy` but use all of it as link text: `{@link #windowedBy(Windows) windowedBy(windows).count()}`

Not perfect, but IMHO still better than no link. Also, linking to `windowBy` seems natural as it's the next method one needs to call instead of deprecated `count()`.